### PR TITLE
Tweaked Some Styling In Sign Up Form

### PR DESCRIPTION
**Pull request by mayank2424**
Changes: Tweaked some changes in input tag and some text-enhancement

### CODE REVIEW

**Comment by rossoskull** [comment]
Please provide a more descriptive title, and try to squash multiple commits in a single commit.

**Comment by Akshat-Jain** [comment]
@mayank2424 Please provide a more suitable description of this PR as already asked by @rossoskull. Please follow up on this. Thanks :-)

**Comment by mayank2424** [comment]
Okay sure sir ,I will do it today

**Comment by fragm3** [comment]
Merge Commits, Provide surge link.

**Comment by mayank2424** [comment]
Okay I will provide

**Comment by mayank2424** [comment]
Hello, I have Resolves the conflict, I  guess there was a text related issue that I have solved and please someone please suggest me good open source projects related to web Development as I am a beginner in open source

**Comment by tabesin** [comment]
Ready to be merged.

File: src/components/Auth/SignUp/SignUp.css [hunk 1]
```diff
@@ -33,13 +33,19 @@
...
+  font-weight:600;
+}
+.app-body-div{
   height: 80%;
   width : 50%;
   float : left;
   background-color : #EEE;
 }
 
+
```
Comment (samagragupta) [inline src/components/Auth/SignUp/SignUp.css]: Remove extra line  . 
Also resolve the merge conflicts and squash the commits as asked by @rossoskull .

**Comment by mayank2424** [comment]
Okay sir I will do that

**Comment by mariobehling** [comment]
@mayank2424 Thanks, please follow best practices:
* add the issue that you are solving
* provide a link to a test setup

**Comment by Orbiter** [comment]
please resolve conflicts

**Pull request merged**